doc/bn.tex

```diff
@@ -826,6 +826,55 @@ \subsection{Long Long Constants}
 
 This will return the 64 least significant bits of the mp\_int $a$.
 
+\subsection{Floating Point Constants}
```

Comment:
:+1: thanks for the docs! great!

Reply:
Would you do me a favour and check it? I don't want another one of this quite embarrassing discussions, you know which one ;-)

There are still some functions undocumented